src/Api/Client.php

```diff
@@ -127,10 +124,43 @@ public function call($endPoint, $httpRequestMethod, $content = [])
 		return $response;
 	}
 
+    /**
+     * @param string
+     * @return resource|false
+     */
+    private function getCurlHandle($endPoint)
+    {
+        $url = self::URL.$endPoint;
+        if (array_key_exists($endPoint, $this->curlHandles)) {
+            if (!isset($this->curlHandles[$endPoint])) {
+                $this->curlHandles[$endPoint] = curl_init($url);
+            }
+            
+            return $this->curlHandles[$endPoint];
+        }
+        
+        return false;
```

Comment:
NULL would be a better return value in case of a failure but generally I would here implement an Exception because if the `curl_init()` fails the process will break anyway.

Reply:
done